### ISSUE TITLE
kvoptions

### ISSUE THREAD

**Issue by FlixXIV** [tudace/tuda_latex_templates#105]
Hi,

thanks for your help. I already tried to install the "kvoptions" -package with miktex package-manager, but there is no so called "kvoptions" -package. I updated the database and tried reinstall of miktex but i can't find it.

**Comment by TeXhackse** [comment]
I see. Just realized that MikTeX current atm does not contain kvoptions. (Source: https://ctan.org/pkg/kvoptions)

Again: This it no change done by us.

This might be due to a remain. I guess it will be added again. Because some really basic packages (as hyperref) rely on it.

During waitiing you can add it manually. Or disable hyperref during that time.

Manual installation is possible using the CTAN source. Explanation can be found here: https://tex.stackexchange.com/questions/2063/how-can-i-manually-install-a-package-on-miktex-windows .

**Comment by open711** [comment]
@TeXhackse Thank you very much for noticing.

I solved the problem on Sunday by completely re-installing MiKTeX and Texmaker. 
I believe that something went wrong with the Oberdiek package https://ctan.org/ctan-ann/id/mailman.1835.1575146287.2586.ctan-ann@ctan.org
In the oberdiek package an updated kvoptions was included. But as there used to be a single kvoptions package MikTex got mixed up which one to use (the latest oberdiek-included vs. the old kvoptions-single). By deinstalling everything I made sure the old single kvoptions was not available.
